staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -398,11 +398,35 @@ func (u *Unstructured) SetClusterName(clusterName string) {
 	u.setNestedField(clusterName, "metadata", "clusterName")
 }
 
-func (u *Unstructured) GetLastApplied() map[string]string {
-	val, _, _ := NestedStringMap(u.Object, "metadata", "lastApplied")
-	return val
+func (u *Unstructured) GetLastApplied() map[string]runtime.RawExtension {
+	m, found, err := nestedMapNoCopy(u.Object, "metadata", "lastApplied")
+	if !found || err != nil {
+		return nil
```

Comment:
Shouldn't we return the error here?

Reply:
I didn't change the file here, so I guess this review might be for @jennybuckley ?